### PR TITLE
Offline metadata

### PR DESCRIPTION
**Pull request by brunobar79**
- Added a new node script to run on CI and extract the colors and dimensions of all the fallback images (~6k)
- Look up the offline version before using rn-dominant-color, which creates additional requests and CPU overhead
- Wipe old metadata from redux (since now most of it will be offline)

Fixes RNBW-1155

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
lgtm, migration is 👌

**Review by osdnk — APPROVED** [review]

**Review by jinchung — APPROVED** [review]
🍔 🍟 doooope thank you!!

File: scripts/extract-metadata.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      allItems[imageUrl] = metadata;
+    }
+  }
+
+  const data = JSON.stringify(allItems);
+  writeFileSync('tokens-metadata.json', data);
+
+  const end = Date.now();
+  const time = (end - start) / 1000;
+  console.log(`Processed ${tokenAddresses.length} images`);
```
[the remark this answers is not in the document]
Comment (brunobar79) [reply scripts/extract-metadata.js:65]: Not part of the app. just an isolated script so console is fine.

File: scripts/extract-metadata.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  console.log(`Took ${time} seconds`);
```
[the remark this answers is not in the document]
Comment (brunobar79) [reply scripts/extract-metadata.js:66]: same

**Pull request merged**